engines/libaio.c

```diff
@@ -51,13 +67,36 @@ static struct fio_option options[] = {
 		.category = FIO_OPT_C_ENGINE,
 		.group	= FIO_OPT_G_LIBAIO,
 	},
+#ifdef FIO_HAVE_IOPRIO_CLASS
+#ifndef FIO_HAVE_IOPRIO
+#error "FIO_HAVE_IOPRIO_CLASS requires FIO_HAVE_IOPRIO"
+#endif
+	{
+		.name	= "prio_percent",
+		.lname	= "prio percentage",
+		.type	= FIO_OPT_INT,
+		.off1	= offsetof(struct libaio_options, prio_percent),
+		.minval	= 1,
+		.maxval	= 100,
+		.help	= "Split the prioclass setting for IO",
```

Comment:
There was kind of a rename a while ago to call it I/O rather than IO.

Reply:
Ok, I'll update this. I'll also update the help text to be accurate, since prioclass and prio_percent cannot be used together in this implementation.